etc/scripts/infer.js

```diff
@@ -16,12 +16,11 @@
  *
```

Comment:
I believe ESLint only lints files ending in `.js` by default. There's probably a way to configure it to lint files with all extensions specifically in the `scripts` folder, but since they're JS files, I thought it made more sense to just rename them for consistency.

Reply:
It appears the only way to lint files without extensions is to manually pass them into eslint as a command-line option, which I did. ESLint [doesn't seem to explicitly support this](https://github.com/eslint/eslint/issues/4071), so it may be worth revisiting this in the future.